ZipBuilder/Sources/ZipBuilder/CocoaPod.swift

```diff
@@ -93,49 +93,8 @@ public enum CocoaPod: String, CaseIterable {
   /// of frameworks get pulled in.
   public func duplicateFrameworksToRemove() -> [String] {
     switch self {
-    case .mlVision:
-      return ["BarcodeDetector.framework",
-              "FaceDetector.framework",
-              "LabelDetector.framework",
-              "TextDetector.framework"]
-    case .mlVisionBarcodeModel:
-      return ["FaceDetector.framework",
-              "GTMSessionFetcher.framework",
-              "GoogleMobileVision.framework",
-              "LabelDetector.framework",
-              "Protobuf.framework",
-              "TextDetector.framework"]
-    case .mlVisionFaceModel:
-      return ["BarcodeDetector.framework",
-              "GTMSessionFetcher.framework",
-              "GoogleMobileVision.framework",
-              "LabelDetector.framework",
-              "Protobuf.framework",
-              "TextDetector.framework"]
-    case .mlVisionLabelModel:
-      return ["BarcodeDetector.framework",
-              "FaceDetector.framework",
-              "GTMSessionFetcher.framework",
-              "GoogleMobileVision.framework",
-              "Protobuf.framework",
-              "TextDetector.framework"]
-    case .mlVisionTextModel:
-      return ["BarcodeDetector.framework",
-              "FaceDetector.framework",
-              "GTMSessionFetcher.framework",
-              "GoogleMobileVision.framework",
-              "LabelDetector.framework",
-              "Protobuf.framework"]
-    case .mlVisionAutoML:
-      return ["BarcodeDetector.framework",
-              "FaceDetector.framework",
-              "LabelDetector.framework",
-              "TextDetector.framework"]
-    case .mlVisionObjectDetection:
-      return ["BarcodeDetector.framework",
-              "FaceDetector.framework",
-              "LabelDetector.framework",
-              "TextDetector.framework"]
+    case .mlVisionBarcodeModel, .mlVisionFaceModel, .mlVisionLabelModel, .mlVisionTextModel:
```

Comment:
Six cases to four?

Reply:
Yep, `. mlVision` and `.mlVisionAutoML` didn't have `Protobuf` or `GTMSessionFetcher` in the list.